python/src/GaussKronrod_doc.i.in

```diff
@@ -15,29 +15,62 @@ Notes
 The Gauss-Kronrod algorithm enables to approximate the definite integral:
 
 .. math::
+    :label: initIntegral
 
     \int_{a}^b f(t)\di{t}
 
 
-with :math:`f: \Rset \mapsto \Rset^p`, using both approximations : Gauss and Kronrod ones defined by:
+with :math:`f: \Rset \mapsto \Rset^p`, using both Gauss and Kronrod approximations, using the rule
```

Comment:
Shouldn't we use the `\outputDim` macro here, instead of p?

Reply:
yes